src/Robot-Dobot_MG400/main.py

```diff
@@ -195,10 +203,21 @@ def deinitialize(self):
 
     def configure(self):
 
+        self.movement_height = float(self.movement_height)
         self.set_acceleration_linear(self.acceleration_factor)  # Linear acceleration factor 1-100
         self.set_speed_global(self.global_speed_factor)  # Global speed factor 1-100
         self.set_speed_linear(self.speed_factor)  # Linear speed factor 1-100
-        self._last_xyzr = (None, None, None, None)
+        # self._last_xyzr = (None, None, None, None)
+        self._last_xyzr = self.get_position()
+
+    def unconfigure(self):
+
+        if self.use_jump_mode:
+            self.move_linear(self._last_xyzr[0], self._last_xyzr[1], self.movement_height, self._last_xyzr[3])
+            self.sync(self.reach_position_timeout)
```

Comment:
This could go into a `move_up()` function, seems the same lines are used in apply() as well. For symmetry reasons a `move_down()` function could be provided as well. It's not needed in the driver, but could be used in actions in the future?

Reply:
It would be something like `move_to_movement_height()`. Whether this is a move up or a move down depends on the start height of the move as you already mentioned in your other comment.

I will add no function for now, but after more testing, it might be worth to add it. A good action would be "go_home" which I just added.